src/skmultiflow/trees/nodes/inactive_learning_node_perceptron_multi_target.py

```diff
@@ -0,0 +1,84 @@
+import numpy as np
+from skmultiflow.trees.nodes import InactiveLearningNodePerceptron
+from skmultiflow.utils import check_random_state
+from skmultiflow.utils.utils import get_dimensions
+
+
+class InactiveLearningNodePerceptronMultiTarget(InactiveLearningNodePerceptron):
```

Comment:
Please add a short description.

Reply:
Documentation updated in commit [5b724c9](https://github.com/scikit-multiflow/scikit-multiflow/pull/149/commits/5b724c98a7bcc2d270f1034185b2978201235576).